README.md

```diff
@@ -23,6 +23,7 @@ void setup(void)
     Serial.begin(115200);
 
     Wire.begin();
+    // Wire.setClock(400000); // set maximum sensor readout speed (400kHz)
```

Comment:
No need to change here

Reply:
My idea here was to give people a hint that you can operate the sensor much faster than with the default. It took me a while to figure that out so I thought a hint would help others.

Reply:
Thanks, I think it's useful but it's not related to the main purpose of this pull request. So it's better to separate the pull request!